blog/templates/blog/base.html

```diff
@@ -12,7 +12,12 @@
 
   <body>
     <div class="page-header">
-        <h1><a href="/">Django Girls Blog</a></h1>
+      {% if user.is_authenticated %}
+      <a href="{% url 'post_new' %}" class="top-menu">
+        <span class="glyphicon glyphicon-plus"></span>
+      </a>
+      {% endif %}
+      <h1><a href="/">Django Girls Blog</a></h1>
```

Comment:
Could you please integrate i18n (`{% trans %}`/gettext) into all strings in your app?

Reply:
@webknjaz done, but I dont know if it is correct